avocado/utils/linux.py

```diff
@@ -72,3 +78,21 @@ def enable_selinux_enforcing():
     if is_selinux_enforcing():
         return True
     return False
+
+
+def is_os_secureboot_enabled():
+    """
+    Check whether the secure-boot is enabled at os level.
+    Check for "00000002" in "/proc/device-tree/ibm,secure-boot" file
+    If found, then secure-boot is enabled.
+
+    :return: True if secureboot is enabled, False if otherwise
+    """
+    try:
+        cmd = "lsprop /proc/device-tree/ibm,secure-boot"
+        for line in process.system_output(cmd).decode("utf-8").splitlines():
+            if "00000002" in line:
+                return True
+    except Exception:
```

Comment:
I am not sure if `Exception` is not too general. When `lsprop` is not in the system, the `process.system_output(cmd) will throw `FileNotFoundError` am I right?